opennms-services/src/main/java/org/opennms/netmgt/poller/PollerEventProcessor.java

```diff
@@ -726,6 +726,8 @@ private void serviceReschedule(Long nodeId, String nodeLabel, Event sourceEvent,
         for (final Service databaseService : databaseServices) {
             if (polledServices.contains(databaseService)) {
                 LOG.debug("{} is being skipped. Already scheduled.", databaseService);
+                //FIXME Discuss with Jesse
```

Comment:
Let's discuss. It looks like I may have forgot the continue statement when I made the changes in NMS-7761.